src/test/java/com/crowdin/cli/commands/actions/DownloadActionTest.java

```diff
@@ -87,13 +87,14 @@ public void testEmptyProject() throws ResponseException, IOException {
             }));
 
         NewAction<PropertiesWithFiles, ProjectClient> action =
-            new DownloadAction(files, false, null, false, null, false, false, false, false);
+            new DownloadAction(files, false, null, false, null, false, false, false, false, false);
         action.act(Outputter.getDefault(), pb, client);
 
         verify(client).downloadFullProject();
         verify(client).startBuildingTranslation(eq(buildProjectTranslationRequest));
-        verify(client).downloadBuild(eq(buildId));
-        verifyNoMoreInteractions(client);
+        when(client.downloadBuild(eq(buildId)))
+                .thenThrow(new RuntimeException());
```

Comment:
Interesting... We can't have an exception here just without any conditions. Does it mean that exceptions will be always present?